src/analytics/build.sbt

```diff
@@ -6,14 +6,14 @@ dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-core" % "2.6.7"
 dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % "2.6.7"
 dependencyOverrides += "com.fasterxml.jackson.module" % "jackson-module-scala_2.11" % "2.6.7"
 
-// dependencyOverrides += "org.json4s" % "json4s-jackson" % "3.2.11"
-// dependencyOverrides += "org.json4s" % "json4s-core" % "3.2.11"
-// dependencyOverrides += "org.json4s" % "json4s-ast" % "3.2.11"
-
-
 libraryDependencies ++= Seq(
+  // Deal with GeoMesa dependency that breaks Spark 2.2
+  "org.json4s" %% "json4s-native" % "3.2.11",
+  "org.json4s" %% "json4s-core" % "3.2.11",
+  "org.json4s" %% "json4s-ast" % "3.2.11",
+
   decline,
-  hive % "provided",
+  sparkHive,// % "provided",
```

Comment:
dropping `% provided` breaks the ability to run this on a cluster.

Reply:
true, sorry shouldn't have committed that